exporter/trace/trace_proto.go

```diff
@@ -47,12 +47,14 @@ const (
 	URLAttribute        = "http.url"
 	UserAgentAttribute  = "http.user_agent"
 	StatusCodeAttribute = "http.status_code"
+	ServiceAttribute    = "service.name"
 
 	labelHTTPHost       = `/http/host`
 	labelHTTPMethod     = `/http/method`
 	labelHTTPStatusCode = `/http/status_code`
 	labelHTTPPath       = `/http/path`
 	labelHTTPUserAgent  = `/http/user_agent`
+	labelService        = `g.co/gae/app/module`
```

Comment:
Not sure if this is a good idea to support this label across all resources - this seems specific to AppEngine?

Reply:
Actually, the mapping seems reverse (i.e. when this label is present, it will be translated to `service.name` on Cloud Monitoring side), which is okay probably.

Reply:
Seems worth adding a comment so that the next person reading this code doesn't have the same question.

Reply:
Comment Added